### ISSUE TITLE
Dropdown kebab button label

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#448]
The Dropdown component examples of a Kebab should include `aria-label` set to something like "Actions".

Updates needed for this issue:
- support the addition of an `aria-label` attribute in the Dropdown handlebar file.
- update the two Kebab examples to define a text string for the `aria-label` attribute, like "Actions"
- include a row in the Accessibility documentation table explaining when `aria-label` would be used.

For example:
```
  <button id="dropdown-example-kebab" class="pf-c-dropdown__toggle" aria-label="Actions" aria-haspopup="true" aria-expanded="true">
    <i class="fas fa-ellipsis-v"></i>
  </button>
```

**Comment by jgiardino** [comment]
@mcarrano Could you provide feedback on what the text string should be for the Kebab example, e.g. "Actions" or "More Actions"? And is it important to be consistent with PF3 examples, where "More Actions" is used?

**Comment by mcarrano** [comment]
It's a bit hard to answer this @jgiardino because sometimes the kabob is used in conjunction with one or two action buttons (to expose more actions) and in other places it exists on it's own. If we must have one label, then I would recommend "Actions" as it works in either case. What are you looking at for PF 3 guidance?

**Comment by jgiardino** [comment]
@mcarrano When discussing with @tlabaj and others, we expect consumers would want to customize this label. For the PF4 core examples, we weren't sure what to use (in the example I included in the description I used just "Actions" but wanted to check with you on that). And in PF4 react, we talked about defining a default value, and were wondering what the default label should be. 

@seanforyou23 had also raised the question about whether consistency between PF3 and PF4 examples is a concern.

**Comment by mcarrano** [comment]
@jgiardino If we all agree that Actions is the best default, then I'm not so concerned about the consistency  with PF 3.

**Comment by jgiardino** [comment]
I'm not strongly tied to "Actions". The only reason I went with that was because "More Actions" seems odd when the kebab is the only component in the example.

**Comment by mcarrano** [comment]
Agree.
